docs/docsite/rst/user_guide/collections_using.rst

```diff
@@ -17,6 +17,16 @@ You can install and use collections through `Ansible Galaxy <https://galaxy.ansi
 Installing collections
 ======================
 
+.. _galaxy_server_config:
+
+Configuring the ``ansible-galaxy`` client
+------------------------------------------
+
+.. include:: ../shared_snippets/galaxy_server_list.txt
```

Comment:
Can we update the snippet to use the real automation hub URLs?

They should be
```
url=https://cloud.redhat.com/api/automation-hub/
auth_url=https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token
```

Reply:
Updated in 95d3a456bb08b198564bb3e0232a2aa45f286db2.